public/app/features/explore/Logs/LogsTableMultiSelect.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { useTheme2 } from '@grafana/ui/src';
+
+import { LogsTableNavColumn } from './LogsTableNavColumn';
+import { fieldNameMeta } from './LogsTableWrap';
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    sidebarWrap: css({
+      overflowY: 'scroll',
+      height: 'calc(100% - 50px)',
+    }),
+    columnHeader: css({
+      fontSize: theme.typography.h6.fontSize,
+      background: theme.colors.background.secondary,
+      position: 'sticky',
+      top: 0,
+      left: 0,
+      padding: '6px 6px 6px 15px',
+      zIndex: 3,
+      marginBottom: theme.spacing(2),
+    }),
+  };
+}
+
+export const LogsTableMultiSelect = (props: {
+  toggleColumn: (columnName: string) => void;
+  filteredColumnsWithMeta: Record<string, fieldNameMeta> | undefined;
+  columnsWithMeta: Record<string, fieldNameMeta>;
+}) => {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  return (
+    <div className={styles.sidebarWrap}>
+      {/* Sidebar columns */}
+      <>
+        <div className={styles.columnHeader}>Common columns</div>
```

Comment:
Using `Common columns` could be confusing as in relation to logs, we use the word `common` -> `common labels/fields` as fields that have the same value for all fields. And here, it is used to describe fields that have value, but it can be any value. 

For me personally, having % is enough to distinguish fields that are 100% filled and fields where sometimes value is missing. I think this is slightly redundant and we are losing valuable space.

Reply:
Yeah the Common and Available are kinda confusing, I've added this to the product DNA doc for topics to review with UX.